sdks/python/apache_beam/coders/standard_coders_test.py

```diff
@@ -116,7 +116,7 @@ def quote(s):
 
 
 STANDARD_CODERS_YAML = os.path.join(
-    os.path.dirname(__file__), 'standard_coders.yaml')
+    os.path.dirname(__file__), '..', 'tests/data/standard_coders.yaml')
```

Comment:
I think you need to os.path.join all the path components, including "tests" and "data" separately.